src/ompl/geometric/planners/rrt/RRTstar.h

```diff
@@ -450,6 +469,15 @@ namespace ompl
             /** \brief Option to use informed sampling */
             bool useInformedSampling_;
 
+            /** \brief Option to use intelligent sampling */
+            bool                                           useIntelligentSampling_;
+
+            /** \brief The ratio of bias towards the beacons */
+            double                                         biasingRatio_;
+
+            /** \brief Flag true after first path is found */
+            bool                                            pathFound_;
```

Comment:
This shouldn't be *strictly* necessary, a solution is found if

1. `opt_->isFinite(bestCost_) == true`, or
2. ` goalMotions_.size() > 0`

If it is desirable to keep this state so explicitly stored (which it may be), then this variable needs to be always maintained. You're only setting it when RRT\*-Smart is in use, which is really confusing.